crmsh/sbd.py

```diff
@@ -345,6 +349,7 @@ def _get_sbd_device_interactive(self):
 
         configured_dev_list = self._get_sbd_device_from_config()
         if configured_dev_list and not bootstrap.confirm("SBD is already configured to use {} - overwrite?".format(';'.join(configured_dev_list))):
```

Comment:
The message here is not really clear about what's already configured and what's going to be overwritten. Could be sysconfig, sbd on-disk metadata, and/or sbd resource and the cluster property settings in pacemaker?

Reply:
Recap the commit title, how about,

[Fix: sbd: not overwrite SYSCONFIG_SBD and sbd-disk-metadata if input 'n'(bsc#1194870)]

INFO: SBD is already configured to use xxxxxx - overwrite (y/n)? 

If 'n', the behavior will be
- bootstrap reuses SYSCONFIG_SBD and tries to reuse sbd-disk-metadata if get initialized already.
- That says, bootstrap skips refreshing a new SYSCONFIG_SBD, and skips 'sbd create' to initialize the sbd disk metadata if exist.